src/Pokemon.java

```diff
@@ -0,0 +1,59 @@
+class Pokemon {
```

Comment:
Misschien had je deze klasse abstract kunnen maken, nu kan je in de main nog wel pokemons aan maken. 

Verder had je hier misschien type bij kunnen zetten dan had je wat meer flexabiliteit gehad binnen de andere klasses!